lib/shared/addon/components/cluster-driver/driver-import/component.js

```diff
@@ -62,6 +66,23 @@ export default Component.extend(ClusterDriver, {
 
       set(this, 'nodeForInfo', node);
     },
+
+    promptUpgradeWarning(cb){
+      if (!get(this, 'needsUpgradeWarning')){
+        this.driverSave(cb)
+
+        return
+      }
+      const { modalService } = this;
+      const { isK3sCluster, isRke2Cluster } = this;
+
+      modalService.toggleModal('modal-confirm-imported-upgrade', {
+        finish: this.confirmUpgrade.bind(this),
+        btnCB:  cb,
+        isK3sCluster,
+        isRke2Cluster
```

Comment:
It doesn't matter much so feel free to ignore but I noticed isRke2Cluster isn't being used within the modal in case you wanted to remove it.